src/Service/FilterService.php

```diff
@@ -25,52 +26,85 @@ public function __construct(EntityManagerInterface $entityManager)
      * @param string $subject
      * @param string $group
      * @param string $student
+     * @param bool $isStationary
+     * @param string $form
+     * @param DateTimeInterface $start
+     * @param DateTimeInterface $finish
      * @return array
      */
-    public function filter(string $faculty = '', string $teacher = '', string $room = '', string $subject = '', string $group = '', string $student = ''): array
+    public function filter(string              $faculty = '',
+                           string              $teacher = '',
+                           string              $room = '',
+                           string              $subject = '',
+                           string              $group = '',
+                           string              $student = '',
+                           bool                $isStationary = null,
```

Comment:
This line gives a warning at the top of the page about deprecated code.
`Deprecated: App\Service\FilterService::filter(): Implicitly marking parameter $isStationary as nullable is deprecated, the explicit nullable type must be used instead in (...)/plan-zajec-zut/src/Service/FilterService.php on line 35
`
Should be changed to:
`?bool                $isStationary = null,`
so the parameter can be either a boolean value or null and it will resolve the warning.